flink-end-to-end-tests/test-scripts/test_sql_client.sh

```diff
@@ -262,10 +271,66 @@ EOF
 JOB_ID=$($FLINK_DIR/bin/sql-client.sh embedded \
   --jar $KAFKA_SQL_JAR \
   --jar $JSON_SQL_JAR \
-  --jar $ELASTICSEARCH_SQL_JAR \
+  --jar $ELASTICSEARCH6_SQL_JAR \
   --jar $SQL_TOOLBOX_JAR \
   --environment $SQL_CONF \
-  --update "$SQL_STATEMENT_5" | grep "Job ID:" | sed 's/.* //g')
+  --update "$SQL_STATEMENT_3" | grep "Job ID:" | sed 's/.* //g')
 
 # 3 upsert results and 6 append results and 3 match_recognize results
 verify_result_line_number 12 "$ELASTICSEARCH_INDEX"
+
+###############################################################################
+# Test sql-connector-elasticsearch7
+###############################################################################
+echo "Executing SQL: Values -> Elasticsearch7 (upsert)"
+
+# prepare client_conf and sql statement
+ELASTICSEARCH_VERSION=7
+ELASTICSEARCH7_MAC_DOWNLOAD_URL='https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.5.1-darwin-x86_64.tar.gz'
+ELASTICSEARCH7_LINUX_DOWNLOAD_URL='https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.5.1-linux-x86_64.tar.gz'
+
+SQL_CONF_CONTENT=$(cat ${SQL_CONF})
+cat >> $SQL_CONF << EOF
+${SQL_CONF_CONTENT//version: 6/version: 7}
```

Comment:
We can use "$ELASTICSEARCH_VERSION" in SQL_CONF to avoid content replacement.

Reply:
nice tips